packages/SwingSet/src/kernel/vatManager/supervisor-subprocess-xsnap.js

```diff
@@ -12,7 +12,6 @@ const decoder = new TextDecoder();
 
 // eslint-disable-next-line no-unused-vars
 function workerLog(first, ...args) {
-  // @ts-ignore
```

Comment:
I guess this dates back before the `console.log` below was commented out.

Here's hoping for a logging configuration API that doesn't involve editing source code :)